src/main/notifications/dnd-linux.ts

```diff
@@ -9,6 +9,9 @@ const GNOME_READ_DND = 'gsettings get org.gnome.desktop.notifications show-banne
 
 function getLinuxDoNotDisturb() {
     try {
+        if (process.platform !== 'linux') {
```

Comment:
Why is this check needed? `getDoNotDisturb` which is the "public" API that runs internally this code means `getLinuxDoNotDisturb` is only ran when `process.platform` is `linux`.

Reply:
0/5 on keeping this or not. It does act as a guard, but really only if someone manually overrides their `process.platform` somehow.

Reply:
yep, 0/5 on that too, so resolving the convo.